Add Bootstrap classes to Markdown tables

emilmelnikov opened the pull request:
Markdown tables are generated without any classes, but Bootstrap
requires adding `.table` classes in order for a table to be styled.

Closes https://github.com/ilastik/ilastik.github.io/issues/151

emilmelnikov commented:
For some reason, a lot of lines in the changed documents had trailing whitespaces, so please ignore whitespace changes in the diff (`git diff -w` or check a checkbox in the GitHub diff UI).

k-dominik reviewed (commented):
This looks cool! Awesome, the only issue I found is the following:

![table](https://user-images.githubusercontent.com/24434157/96888056-8173fc00-1485-11eb-8c15-8e9bbd84d1cf.png)

is it possible to wrap those texts there (somehow enforce the table to the content width?)

Also could you extract the whitespace changes into a separate commit?

emilmelnikov commented:
> is it possible to wrap those texts there (somehow enforce the table to the content width?)

Only if some text is out of the `<pre>` elements.

I've changed the text and formatting, but I'm not sure what the original documentation author meant (that is, where are `identifiers_in_actual_data` and where *their descriptions*). @k-dominik could you make sure that the meaning is the same?

k-dominik commented:
Hi @emilmelnikov , the table looks great! As for the _Object IDs_ in that table: There are actually _no_ identifiers in the data (I checked :o, [e.g. here](https://github.com/ilastik/ilastik/blob/master/ilastik/applets/tracking/annotations/annotationsGui.py#L1898) in case you want to make your eyes bleed). So this is more or less arbitrary. Maybe rename the `Object IDs` column in the table to `Columns`?

emilmelnikov commented:
I also added a missing `MultiFrameMoves` dataset, and added a note that we also have energy datasets.

![image](https://user-images.githubusercontent.com/1649961/99491859-d31c8300-296c-11eb-9775-790ff7ee3fb3.png)

k-dominik commented:
that's cool. Could you check whether it is actually possible to generate these fields in export files? I have no clue how MultiFrameMoves are supposed to happen. (Quickly tried, but didn't manage to generate one).

emilmelnikov commented:
> Could you check whether it is actually possible to generate these fields in export files? I have no clue how MultiFrameMoves are supposed to happen.

I don't really understand very well what is going on in the manual tracking code, but it seems that multi-frame merges happen on this condition: https://github.com/ilastik/ilastik/blob/a7ee78d7936ed573b10c6ff4d8733e1f126a00ea/ilastik/applets/tracking/manual/manualTrackingGui.py#L938

> Also about the energies: I would not know where these should come from in manual tracking...

Yes, I looked at the code, and in manual tracking we always write 0 energy for each event.

k-dominik reviewed (approved):
:+1: the tracking docs finally look somewhat bearable on mobile devices...
